### PR TITLE
Artifacts Age 2 : Holy grail, babylonian chronicles, and terra cotta army #88

### PR DESCRIPTION
**Pull request by ultimatefiend**
Includes 3 new cards: Holy grail, babylonian chronicles, and terra cotta army

It includes a fix for electrum (age 1) as well.

### CODE REVIEW

File: innovation.game.php [hunk 1]
```diff
@@ -9750,6 +9776,91 @@ function stInteractionStep() {
...
+                'player_id' => $player_id,
+                'n' => 1,
+                'can_pass' => false,
+                
+                'owner_from' => $player_id,
+                'location_from' => 'hand',
+                'owner_to' => $player_id,
+                'location_to' => 'score',
+                
+                'without_icon' => 4 // tower
```
Comment (micahstairs) [inline innovation.game.php:9860]: Don't forget to add a line for 'score_keyword''! Or else this score won't count towards Monument.

**Review by micahstairs — CHANGES_REQUESTED** [review]
Nice work!!

File: innovation.game.php (hunk 1 above)
Comment (micahstairs) [reply innovation.game.php:9861]: Thanks!

Comment (ultimatefiend) [reply innovation.game.php:9861]: done

**Review by micahstairs — APPROVED** [review]
I'll get this merged in as soon as I get the chance!

**Pull request merged**